
pub fn do_assignment_2() -> Result<(), Box<dyn std::error::Error>> {

    Ok(())
}